Makefile

```
@@ -35,3 +35,6 @@ ocp_cleanup:
 
 host_cleanup:
 	./host_cleanup.sh
+
+bell:
+	echo "Done!" $$'\a'
```

This doesn't work for me. How about we just use tput?
```suggestion
	echo "Done!"
	tput bel
```

---

An earlier version of this PR did use tput, and that broke in CI because $TERM was not set.